chemdataextractor/parse/table.py

```diff
@@ -278,11 +286,13 @@ def interpret(self, result, start, end):
         solvent = first(result.xpath('./name/text()'))
         if solvent is not None:
             context = {'solvent': solvent}
-            c.melting_points = [MeltingPoint(context)]
-            c.quantum_yields = [QuantumYield(context)]
-            c.fluorescence_lifetimes = [FluorescenceLifetime(context)]
-            c.electrochemical_potentials = [ElectrochemicalPotential(context)]
-            c.uvvis_spectra = [UvvisSpectrum(context)]
+	    # RBT (Added ** to context)
+            c.melting_points = [MeltingPoint(**context)]
+            c.glass_transitions = [GlassTransition(**context)]
+            c.quantum_yields = [QuantumYield(**context)]
+            c.fluorescence_lifetimes = [FluorescenceLifetime(**context)]
+            c.electrochemical_potentials = [ElectrochemicalPotential(**context)]
+            c.uvvis_spectra = [UvvisSpectrum(**context)]
```

Comment:
Good catch on the missing `**` - I forgot to update this when I changed the model objects to accept keyword arguments rather than a single dictionary. The `**` unpacks the `context` dict to be the equivalent of kwargs like `solvent='acetonitrile` etc.